core/pypika_utils.py

```diff
@@ -77,7 +79,8 @@ def __init__(self, operation=None, operation_steps=None, source=None):
             self.initial_table_name = query_steps.first().source.active_mirror_name
             self.initial_schema_name = query_steps.first().source.schema
 
-        self.current_dataset = Table(self.initial_table_name, schema=self.initial_schema_name)
+        self.current_dataset = Table(
+            self.initial_table_name, schema=self.initial_schema_name)
```

Comment:
You have a lot of these formatting changes across multiple files ... curious, are u doing them manually, or is it some automated linter doing it?

Reply:
If it's a linter, let me know which one it is ... best we agree on these things to avoid clashes, cos if we all use different ones, we're likely to keep pushing changes that don't add new functionality but simply undo each other's code styles ...

Reply:
It's a linter, `pylint`.